sda-auth/dev-server/docker-compose.yml

```diff
@@ -43,6 +43,16 @@ services:
       - CEGA_USERS_USER=dummy
     ports:
       - 8443:8443
+  keygen:
+    image: golang:alpine3.16
+    container_name: keygen
+    command:
+      - "/bin/sh"
+      - "-c"
+      - if [ ! -f "/out/c4gh.sec.pem" ]; then wget -qO- "https://github.com/neicnordic/crypt4gh/releases/latest/download/crypt4gh_linux_x86_64.tar.gz" | tar zxf -;
+        ./crypt4gh generate -n c4gh -p privatekeypass && mv *.pem /out/; fi
+    volumes:
+      - ../keys:/out
```

Comment:
Don't pollute the local directories, use docker volumes for this kind of stuff that are temporary.